azure-pipelines-dependency-graph-testing.yml

```diff
@@ -180,7 +180,9 @@ stages:
       - job: compatibility_matrix
         condition: eq(stageDependencies.scope_check.changes.outputs['CheckChanges.GEChanged'], true)
         variables:
-          GE_pytest_opts: '--no-sqlalchemy'
+          - group: 'common_secrets'
+          - name: GE_pytest_opts
+            value: '--no-sqlalchemy'
```

Comment:
Initially, I commented this `GE_pytest_opts` key in order to get the `coveralls` step working ([which it now does](https://coveralls.io/builds/52134850) 🎊 ), but I assume we need to add this back.

I'm also assuming what I did to add it back is equivalent but LMK if that's not correct.